src/api/api.js

```diff
@@ -10,3 +10,10 @@ export function fetchArtilces() {
     .then((res) => res.data.articles)
     .catch((err) => console.log(err.message));
 }
+
+export function ArtilceDetailAndComments(articleId, comment = "") {
```

Comment:
You've combined two tickets into one here. Can you please separate this, so we have one function just getting the individual articles, and one just getting the comments?